synapse/lib/stormlib/crypto.py

```diff
@@ -0,0 +1,43 @@
+import synapse.exc as s_exc
+import synapse.common as s_common
+
+import synapse.lib.crypto.coin as s_coin
+import synapse.lib.stormtypes as s_stormtypes
+
+@s_stormtypes.registry.registerLib
+class CryptoLib(s_stormtypes.Lib):
+    '''
+    A Storm library which implements helpers for cryptocurrencies.
+    '''
+    _storm_locals = (
+        {'name': 'ether_eip55', 'desc': 'Convert an Ethereum address to a checksummed address.',
+         'type': {'type': 'function', '_funcname': 'eip55',
+                  'args': (
+                      {'name': 'addr', 'type': 'str', 'desc': 'The Ethereum address to be converted.'},
+                  ),
+                  'returns': {'type': 'list',
+                              'desc': 'A list of (<bool>, <addr>) for status and checksummed address.', },
+        }},
+    )
+
+    _storm_lib_path = ('crypto',)
```

Comment:
We should name this `crypto.coin` at a minimum. We don't want to muddy a namespace where we may want to add other cryptographic primitives in the future.

Reply:
renamed to `crypto.coin`, unless you think maybe we need to go to `crypto.coin.ethereum`?

Reply:
sounds good